### ISSUE TITLE
Log Error Integrity constraint violation:

### ISSUE THREAD

**Issue by CreateSean** [verbb/feed-me#387]
### Description
I updated a import with a new xml file (new fields) and now when I run it, I get this error each time I try to run the feed.

```

The Word For You Today: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'id' in where clause is ambiguous The SQL being executed was: SELECT `elements`.`id`, `elements`.`fieldLayoutId`, `elements`.`uid`, `elements`.`enabled`, `elements`.`archived`, `elements`.`dateCreated`, `elements`.`dateUpdated`, `elements_sites`.`slug`, `elements_sites`.`uri`, `elements_sites`.`enabled` AS `enabledForSite`, `users`.`username`, `users`.`firstName`, `users`.`lastName`, `users`.`email`, `users`.`admin`, `users`.`locked`, `users`.`pending`, `users`.`suspended`, `users`.`lastLoginDate`, `users`.`lockoutDate`, `users`.`photoId`, `users`.`hasDashboard`, `content`.`id` AS `contentId`, `content`.`field_eventAgreement`, `content`.`field_alternateTitle`, `content`.`field_audioLink`, `content`.`field_wordAuthor`, `content`.`field_bio`, `content`.`field_bitlyLink`, `content`.`field_callLetters`, `content`.`field_callNumber`, `content`.`field_columnTitle`, `content`.`field_contactEmail`, `content`.`field_contactName`, `content`.`field_eventCopy`, `content`.`field_email`, `content`.`field_eventCost`, `content`.`field_eventLegal`, `content`.`field_facebook`, `content`.`field_fax`, `content`.`field_freeEvent`, `content`.`field_instagram`, `content`.`field_link1Text`, `content`.`field_link2Text`, `content`.`field_link1Link`, `content`.`field_link2Link`, `content`.`field_listenNowFeedUrl`, `content`.`field_listeningRadius`, `content`.`field_managerName`, `content`.`field_eventVenue`, `content`.`field_noResultsMessage`, `content`.`field_offSiteLink`, `content`.`field_onAirAnnouncementConsideration`, `content`.`field_openInNewTab`, `content`.`field_originalDate`, `content`.`field_phone`, `content`.`field_rights`, `content`.`field_seoDescription`, `content`.`field_seoTitle`, `content`.`field_showName`, `content`.`field_showTimes`, `content`.`field_stationCalendar`, `content`.`field_story`, `content`.`field_studioPhone`, `content`.`field_submittedBy`, `content`.`field_summary`, `content`.`field_twitter`, `content`.`field_urlLink`, `content`.`field_venueAccessibility`, `content`.`field_verse`, `content`.`field_verseLink`, `content`.`field_video`, `content`.`field_youtube` FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `elementsSitesId`, `content`.`id` AS `contentId` FROM `elements` `elements` INNER JOIN `users` `users` ON `users`.`id` = `elements`.`id` INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id` INNER JOIN `content` `content` ON `content`.`elementId` = `elements`.`id` WHERE (`elements_sites`.`siteId`='1') AND (`id` = '17') AND (`content`.`siteId`='1') AND (`elements`.`archived`=FALSE) AND (`elements_sites`.`enabled`=TRUE) ORDER BY `users`.`username` LIMIT 1) `subquery` INNER JOIN `users` `users` ON `users`.`id` = `subquery`.`elementsId` INNER JOIN `elements` `elements` ON `elements`.`id` = `subquery`.`elementsId` INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`elementsSitesId` INNER JOIN `content` `content` ON `content`.`id` = `subquery`.`contentId` ORDER BY `users`.`username` - Schema.php:664 | 8/30
-- | --


```

### Steps to reproduce

1. set up feed
2. run feed

### Additional info

- Plugin version: 3.0.0-beta.21
- Craft version: Craft CMS 3.0.21

**Comment by sjcallender** [comment]
+1 @engram-design 

We're getting the same error with Feed Me Pro.

**Comment by benjaminkohl** [comment]
Is there an update on the progress of getting this issue resolved? This is a showstopper for us and I tried rolling back to several previous versions only to find showstopper bugs in those releases as well. In those cases, the control panel pages wouldn't even load due to stray commas in the code and/or incorrect tab URLs.

**Comment by CreateSean** [comment]
I sent in an official support request after purchasing feedme. Got an automatic response that they are on holiday until after the dotall conference.

**Comment by ryanmasuga** [comment]
Already been 19 days since this was posted. Guess we won't be doing any successful imports on our project for another 10 days at least!

**Comment by sjcallender** [comment]
@engram-design Thanks, Josh. It is appreciated. I'll buy you a couple of pretzels in Berlin.

**Comment by sergeypavlikhin** [comment]
+1

**Comment by CreateSean** [comment]
Yes still waiting - a major slow down on a site that is supposed to luanch in the next two weeks.

**Comment by engram-design** [comment]
Sorry all for this issue and delay, fixed in [3.0.0-beta.22](https://github.com/verbb/feed-me/releases/tag/3.0.0-beta.22).
